packages/sources/src/components/input-number/input-number.stories.ts

```diff
@@ -27,7 +27,7 @@ export function storiesOfInputNumber<TemplateFnReturnType>(
       },
       argTypes: inputNumberArgTypes,
       args: {
-        label: 'Default',
+        label: 'Input number',
```

Comment:
Kunnen we hier niet beter een tekst als `Aantal schuurtjes` invullen, oid?

Reply:
Ik zocht ook al naar wat hiervoor eigenlijk 'de lijn' is. Maar die is er nog niet echt. Alleen vond ik 'Default' net ff een tandje te suf.

Reply:
Maar eens dat het inzetten van de componentnaam nou ook niet bepaald verhelderend werkt. Ik zal er iets anders van maken.